src/Resources/Locales/en.json

```diff
@@ -608,6 +608,64 @@
             "libraryName": "Panel",
             "description": "A layout panel"
         },
+        "dataHistory": {
+            "title": "Data history widget",
+            "libraryName": "Data history",
+            "description": "Show timeseries data in context of your 3D scene",
+            "notEnabledDescription": "Data history is not enabled for this ADT instance.",
+            "form": {
+                "connectionStringTitle": "Connection string",
+                "connectionStringPlaceholder": "Enter connection string (e.g. key1=value1;key2=value2;key3=value3)",
+                "connectionStringInformation": "Data history widgets target the Azure Data Explorer instance connected to the Azure Digital Twins instance at the time of creation.",
+                "connectionLoadingText": "Loading connection information...",
+                "noConnectionInformationText": "No connection information found",
+                "displayNamePlaceholder": "Enter display name",
+                "timeSeries": {
+                    "timeSeriesDescription": "Add up to {{maxTimeSeriesCount}} timeseries to display",
+                    "title": "Time series",
+                    "description": "A series to display in the line chart",
+                    "property": "Property",
+                    "unit": "Unit",
+                    "label": "Label",
+                    "labelPlaceholder": "Enter label to show in chart",
+                    "add": "Add time series",
+                    "edit": "Edit time series",
+                    "update": "Update time series",
+                    "remove": "Remove time series"
+                },
+                "chartOptions": {
+                    "yAxisType": {
+                        "description": "Chart option to set if each series has separate (Independent) y-axes or sharing the same y-axis (Shared)",
+                        "label": "Y axis type",
+                        "sharedLabel": "Shared",
+                        "independentLabel": "Independent"
+                    },
+                    "quickTimeSpan": {
+                        "label": "Default time span",
+                        "information": "Default time range selection in local time for the x-axis in chart",
+                        "options": {
+                            "Last 15 mins": "Last 15 mins",
+                            "Last 30 mins": "Last 30 mins",
+                            "Last Hour": "Last Hour",
+                            "Last 3 Hours": "Last 3 Hours",
```

Comment:
nit, we capitalized hours but not mins. I think hours and days and years should all be lower case since we typicially follow sentence casing.